var/spack/repos/builtin/packages/py-botorch/package.py

```diff
@@ -14,15 +14,21 @@ class PyBotorch(PythonPackage):
 
     maintainers("adamjstewart")
 
+    version("0.8.3", sha256="e529f7adbb2b54f46125ae904682fc0f0d02ab8bdb9067ede521c379b355bf73")
     version("0.6.4", sha256="3fd28417f55749501a45378f72cd5ca7614e2e05b7b65c6b4eb9b72378bc665a")
 
     depends_on("python@3.7:", type=("build", "run"))
-    # TODO: replace this after concretizer learns how to concretize separate build deps
+    depends_on("python@3.8:", when="@0.8.3:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    # depends_on('py-setuptools@:47', type='build')
+    depends_on("py-setuptools@:47", when="@:0.6.4", type='build')
```

Comment:
```suggestion
    depends_on("py-setuptools@:47", when="@:0.6.4", type="build")
```
To fix style (not sure why it didn't work)